Linux/sgx/test_app/enclave/TestEnclave.cpp

```diff
@@ -295,130 +293,188 @@ void t_sgxssl_call_apis()
     printf("Start tests\n");
     
     SGXSSLSetPrintToStdoutStderrCB(vprintf_cb);
+    OSSL_PROVIDER *prov;
+#ifndef SGXSSL_FIPS
+    prov = OSSL_PROVIDER_load(NULL, "default");
+#else
+    void *entry = get_ossl_fips_sym("OSSL_provider_init");
```

Comment:
The API is updated to 'sgx_get_ossl_fips_sym'. Please update this and the one in fips_test.

Reply:
Pushed one commit to rename the API.